README.md

```diff
@@ -1,2 +1,91 @@
 # DocToolsBundle
 Helper tools for documentation
+
+## Installation
+
+As usual, there are a few steps required to install this bundle:
+
+1.a **Add this bundle to your project using composer**:
+
+```bash
+$ composer require prestashop/doc-tools-bundle
+```
+
+1.b **Add this bundle to your project as a composer dependency (from your forked repository)**:
+
+```javascript
+    // composer.json
+    {
+        // ...
+        require-dev: {
+            // ...
+            "prestashop/doc-tools-bundle": "dev-my-branch"
+        },
+        // ...
+        "repositories": [
+            // ...
+            {
+                "type": "vcs",
+                "url": "https://github.com/myfork/DocToolsBundle",
+                "canonical": false
+            },
+            // ...
+        ],
+    }
+```
+
+```bash
+$ composer require prestashop/doc-tools-bundle
+```
+
+2. **Add this bundle to your application kernel**:
+
+```php
+    // app/AppKernel.php
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+            new PrestaShop\DocToolsBundle\DocToolsBundle(),
+        );
+
+        return $bundles;
+    }
+```
+
+3. **Configure the bundle via app/config**
+
+```yml
+# app/config/config.yml
+doc_tools:
+  docs_src_path: /root/path/to/docs/project/src
+  cqrs_folder: content/1.7/development/architecture/domain/references
+```
+
+4. **How to use**
```

Comment:
It would great to add a section `5. **Test**` that explains how the tests are built (I have seen you load the Bundle inside a fixture kernel), how to run them